### PR TITLE
Events laura

### PR DESCRIPTION
**Pull request by ldominique**
Creation of wtw_feed and related_feed topics.
Remove of language related field in context
Addition of required field in website context (visitor_group,...)

### CODE REVIEW

File: specs/related_feed.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+| video_display             |Each time a video has been seen by a user either on the player page or at the end of the player. Seen by the user means that it has been visible on the page (above the fold) and not that it has been served by Dailymotion.                                                       |
+| video_click               |Each time a user clicks on a related video          |
+| more_click                |Each time a user click on "Load all videos" at the end of the related list on the player page|
+
+
+### fieds associated to each event
+
+| name                      | type     | format/values | mandatory      | definition    | associated events|
+|---------------------------|----------|---------------|----------------|-----------------------------------------------------------|-------|
+| video_id      | INTEGER  |               | Y              | The main video id currently watched by the user                          |video_display / video_click / more_click |
```
Comment (stephanie-w) [inline specs/related_feed.md]: not INTEGER but string "x123abc" with a xid format that should be defined in the lexicon, idem for related_video_id

File: specs/related_feed.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                "request_id": "70b1172e8f33f825a83a8dbf7920adc769c54100",
+                "related_videos":[
+                    {"related_video_id:x3o9gd4,related_video_position":1}
+                    {"related_video_id:x3o3bk4,related_video_position":2}
+                ]
+                "current_related_algorithm": "no-related",
+                "related_feed_position": "in_player",
+                },
+        "created_ts":1450130802589}]
+    }
```
Comment (stephanie-w) [inline specs/related_feed.md]: If you want to send two events in a once, that's the way to do it:
`
events: [{ one_event_struct},{second_event_struct}]
`

File: specs/website/_context.md [hunk 3]
```diff
@@ -3,7 +3,11 @@
 |-------------------|--------|---------------|----------------|-------------|
 | client_type       | STRING | [client_type](/lexicon.html#client_type)  | Y |  |
 | client_version    | STRING | [client_version](/lexicon.html#client_version)  | Y |  |
+| device            | STRING | | Y |  |An object containing client`s browser and OS information
```
Comment (stephanie-w) [inline specs/website/_context.md]: HTTP user agent? if yes, it's already available in EIN worker

**Comment by ldominique** [comment]
Update done as discussed.

**Pull request merged**